content/docs/bdd/history.md

```diff
@@ -0,0 +1,18 @@
+---
+title: History of BDD
+subtitle: Origins of BDD and Cucumber
+---
+
+Behavior-driven development was pioneered by Daniel Terhorst-North back in the early 00s, as he explained in a [2006 article](http://dannorth.net/introducing-bdd/). It grew from a response to test-driven development (TDD), as a way to help programmers on new agile teams "get straight to the good stuff" of knowing how to approach testing and coding, and minimize misunderstandings. BDD has evolved into both analysis and automated testing at the acceptance level.
+
+# Early tools supporting BDD
+
+In 2003, Daniel Terhorst-North started writing a replacement for JUnit called JBehave, using vocabulary based on "behaviour" rather than "test". Influenced by the idea of ubiquitous language introduced in Eric Evans' [Domain-Driven Design](https://domainlanguage.com/ddd/), and also focusing on business value, he and Chris Matts developed the "Given/When/Then" template to capture a story's accdeptance criteria in an executable form. 
+
+In 2005, the [RSpec](https://rspec.info/) project, which supported BDD in the Ruby language, was founded by [Dave Astels](http://daveastels.com/), [Steven Baker](https://stevenrbaker.com/), Aslak Hellesøy. and [David Chelimsky](https://twitter.com/dchelimsky). 
```

Comment:
```suggestion
In 2005, the [RSpec](https://rspec.info/) project, which supported BDD in the Ruby language, was founded by [Dave Astels](http://daveastels.com/), [Steven Baker](https://stevenrbaker.com/), [Aslak Hellesøy](https://twitter.com/aslak_hellesoy) and [David Chelimsky](https://twitter.com/dchelimsky). 
```

Reply:
@aslakhellesoy Unless you have a website or something to link to instead of Twitter?

Reply:
Well, I didn't link to Aslak since he must be all over this Cucumber doc, I guess that was dumb, what is a good link for him? And I couldn't find a good website for David Chelimsky, do you have one?

Reply:
Or did you put it into the suggestion - I haven't figured out how to see the whole suggestion!